python/lsst/dax/apdb/scripts/create_cassandra.py

```diff
@@ -56,6 +57,9 @@ def create_cassandra(
     config = ApdbCassandra.init_database(ra_dec_columns=ra_dec_list, table_options=options, **kwargs)
     config.save(output_config)
 
+    apdb = ApdbCassandra.from_config(config)
+    apdb.metadata.set("instrument", instrument)
```

Comment:
I think the key should be defined in [`ApdbCassandra`](https://github.com/lsst/dax_apdb/blob/main/python/lsst/dax/apdb/cassandra/apdbCassandra.py#L259-L269), just like the other metadata keys.

There's also the question of how to handle old DBs. The versioning of `ApdbCassandra`/`ApdbSql` seems to _exclude_ the metadata table (e.g., the classes have explicit fallback code for whether or not the table exists), but `dax_apdb_migrate` is otherwise a great way to handle this sort of thing.